xbmc/TextureCache.cpp

```diff
@@ -31,7 +31,7 @@
 using namespace XFILE;
 using namespace std::chrono_literals;
 
-CTextureCache::CTextureCache() : CJobQueue(false, 1, CJob::PRIORITY_LOW_PAUSABLE)
+CTextureCache::CTextureCache() : CJobQueue(false, 2, CJob::PRIORITY_LOW_PAUSABLE)
```

Comment:
Mind explaining the 1 -> 2 change? Probably has to do with the hack that was removed below?

Reply:
Oops! I missed mentioning this in the PR. I changed this to improve performance. `CJobManager` allows 2 workers for this priority and for my test folder the thumbnail creation time dropped from about 350s to 250s on my ryzen 5 laptop. IIRC, memory usage was around 5GB for both.

However, if there is a concern about memory or about increasing load on other systems that Kodi supports, I'll revert this to 1.

Reply:
Yes. Kodi is still commonly used on arm platforms with 2GB or less.

Reply:
IMO changing it back to 1 would be better for the reasons mentioned by @popcornmix

Reply:
Pushing the change. 

Just for the record, with a test folder of 100 images, here is the memory usage as seen in Visual Studio.
2 jobs thumbnail creation: 3.1GB which is stable during the entire duration of the queue
2 jobs after thumbnail creation: 2.8GB 
1 job thumbnail creation: 2.9GB which is stable during the entire duration of the queue
1 job after thumbnail creation: 2.8GB
Fresh start of Kodi, test folder opened with thumbnails created in previous session: 276MB!